src/compile/compile.ts

```diff
@@ -67,6 +67,7 @@ function assemble(model: Model) {
 }
 
 export function assembleRootGroup(model: Model) {
+  const layout = stringValue(model.name(LAYOUT + ''));
```

Comment:
[`stringValue`](https://github.com/vega/vega-util#stringValue) is helpful when including a string value within expression strings. `model.name(LAYOUT) -> 'layout'` whereas `stringValue(model.name(LAYOUT)) -> '"layout"'`. Thus, in the signal expression strings below, I can simply concatenate the `layout` variable without worrying about escaping it, etc.